cclib/parser_properties/atomcoords.py

```diff
@@ -0,0 +1,38 @@
+from cclib.parser_properties.base_parser import base_parser
+from cclib.parser_properties import utils
+import numpy as np
+
+class atomcoords(base_parser):
+    """
+    Docstring? Units?
+    """
+    known_codes = ["gaussian"]
+
+    @staticmethod
+    def gaussian(file_handler, ccdata):
+        # ccdata is "const" here and we don't need to modify it yet. The driver will set the attr
+        line = file_handler.last_line
+        constructed_data = None
+        if line.strip() == "Standard orientation:":
+            line = file_handler.skip_lines(['d','Center','Number','d'],virtual=True)
+            constructed_data=[]
+            atomcoords=[]
+            while list(set(line.strip())) != ["-"]:
+                broken = line.split()
+                atomcoords.append(list(map(float, broken[-3:])))
+                line = file_handler.virtual_next()
+            constructed_data.append(atomcoords)
+            return constructed_data
+        return None
+
+    @staticmethod
+    def parse(file_handler, program:str, ccdata) -> np.ndarray:
+        constructed_data = None
+        if program in atomcoords.known_codes:
+            file_handler.virtual_set()
+            program_parser = getattr(atomcoords, program)
+            constructed_data = program_parser(file_handler, ccdata)
+            file_handler.virtual_reset()
```

Comment:
it feels like this might be a repeated paradigm can we factor this out? nothing to change right now